contrib/pg_tde/expected/default_principal_key.out

```diff
@@ -21,10 +21,10 @@ SELECT id, provider_name FROM pg_tde_list_all_global_key_providers();
 (1 row)
 
 -- Should fail: no principal key for the database yet
-SELECT  key_provider_id, key_provider_name, principal_key_name
-		FROM pg_tde_principal_key_info();
+SELECT  key_provider_id, key_provider_name, key_name
+		FROM pg_tde_key_info();
 ERROR:  Principal key does not exists for the database
-HINT:  Use set_principal_key interface to set the principal key
+HINT:  Use set_key interface to set the key
```

Comment:
Some errors still use principal key while this calls it key.

Reply:
Reverted error messages to use `principal key` as it was before